modules/network-observability-netobserv-cli-reference.adoc

```diff
@@ -1,13 +1,14 @@
 // Automatically generated by './scripts/generate-doc.sh'. Do not edit, or make the NETOBSERV team aware of the editions.
 :_mod-docs-content-type: REFERENCE
+
 [id="network-observability-netobserv-cli-reference_{context}"]
 = Network Observability CLI usage
 
-You can use the Network Observability CLI (`oc netobserv`) to pass command line arguments to capture flow data and packet data for further analysis, enable Network Observability Operator features, or pass configuration options to the eBPF agent and `flowlogs-pipeline`.
+You can use the Network Observability CLI (`oc netobserv`) to pass command line arguments to capture flows data and packets data for further analysis, enable Network Observability Operator features or pass configuration options to the eBPF agent and `flowlogs-pipeline`.
```

Comment:
```suggestion
You can use the Network Observability CLI (`oc netobserv`) to pass command line arguments to capture flows, packets or metrics data for further analysis and enable features supported by Network Observability Operator.
```
I am not sure what does last sentence "or pass configuration options to the eBPF agent and `flowlogs-pipeline`." mean here.

Reply:
Ok I updated to this, since Julien 👍 .